.travis.yml

```diff
@@ -1,14 +1,26 @@
 language: cpp
 matrix:
   include:
-    - name: "ARCH=arm"
-      env: ARCH=arm
-    - name: "ARCH=arm64"
-      env: ARCH=arm64
-    - name: "ARCH=arm64 LD=ld.lld"
-      env: ARCH=arm64 LD=ld.lld-8
-    - name: "ARCH=x86_64"
-      env: ARCH=x86_64
+    - name: "ARCH=arm REPO=linux"
+      env: ARCH=arm REPO=linux
+    - name: "ARCH=arm64 REPO=linux"
+      env: ARCH=arm64 REPO=linux
+    - name: "ARCH=arm64 LD=ld.lld REPO=linux"
+      env: ARCH=arm64 LD=ld.lld-8 REPO=linux
+    - name: "ARCH=x86_64 REPO=linux"
+      env: ARCH=x86_64 REPO=linux
+    - name: "ARCH=arm REPO=linux-next"
+      env: ARCH=arm REPO=linux-next
+      if: type = cron
```

Comment:
neat, where did you find documentation for `if`? I didn't see any on [matrix](https://docs.travis-ci.com/user/customizing-the-build/) or [cron](https://docs.travis-ci.com/user/cron-jobs/).

Reply:
Gah meant to add that to the commit message.

https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

https://docs.travis-ci.com/user/conditions-v1

Reply:
Neat; looks good. Just minor bikeshedding about the name and allowing defaults to be implicit. 👽 🌶